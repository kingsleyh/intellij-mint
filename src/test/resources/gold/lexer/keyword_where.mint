module B {

  fun yay : Void {
    something()
  } where {
    a = Debug.log("woops")
  }

}