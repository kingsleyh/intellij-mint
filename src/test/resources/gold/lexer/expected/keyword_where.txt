module ('module')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('B')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
LOWER_IDENT ('yay')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Void')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
LOWER_IDENT ('something')
( ('(')
) (')')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE (' ')
where ('where')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
LOWER_IDENT ('a')
WHITE_SPACE (' ')
= ('=')
WHITE_SPACE (' ')
CAPITALISED_IDENT ('Debug')
. ('.')
LOWER_IDENT ('log')
( ('(')
STRING_LITERAL ('"woops"')
) (')')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')